webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/marshalling/XmlUnserialisationImpl.scala

```diff
@@ -653,6 +654,24 @@ class GlobalParameterUnserialisationImpl extends GlobalParameterUnserialisation
   }
 }
 
+class SecretUnserialisationImpl extends SecretUnserialisation {
+  def unserialise(entry:XNode) : Box[Secret] = {
+    for {
+      secret      <- {
+        if(entry.label ==  XML_TAG_SECRET) Full(entry)
+        else Failure("Entry type is not a <%s>: %s".format(XML_TAG_SECRET, entry))
+      }
+      fileFormatOk     <- TestFileFormat(secret)
+
+      name             <- (secret \ "name").headOption.map( _.text ) ?~! ("Missing attribute 'name' in entry type secret : " + entry)
+//      value            <- (secret \ "value").headOption.map( _.text ) ?~! ("Missing attribute 'value' in entry type secret : " + entry)
+      description      <- (secret \ "description").headOption.map( _.text ) ?~! ("Missing attribute 'description' in entry type secret : " + entry)
+    } yield {
+      Secret(name, "", description)
```

Comment:
Not the best solution, maybe create a `SecretInfo(name: String, description: String)` used instead of Secret
and transform `Secret(name: String, value: String, description: String)` into  `Secret(value: String, info: SecretInfo)`